cfgov/unprocessed/apps/youth-employment-success/css/main.less

```diff
@@ -31,6 +31,13 @@
   }
 }
 
+.yes-tip {
+  .cf-icon-svg {
+    font-size: 1.25em;
```

Comment:
Probably should use a calculation to make it clearer what size this is aiming for 
```
font-size: unit( 20px / @base-font-size-px, em );
```